app/actions.py

```diff
@@ -1062,28 +1064,26 @@ def stripTrailingWhiteSpace(self):
         self.performDeleteRange(i, found.regs[0][0], i, found.regs[0][1])
 
   def unindent(self):
-    if self.selectionMode == app.selectable.kSelectionAll:
-      self.cursorMoveAndMark(len(self.lines) - 1 - self.penRow, -self.penCol,
-          -self.markerRow, -self.markerCol,
-          app.selectable.kSelectionLine - self.selectionMode)
-      self.redo()
-      self.unindentLines()
+    if self.selectionMode == app.selectable.kSelectionNone:
+      pass
```

Comment:
Oh I asked you about this in #27 so I intentionally left this blank for now. I'll update that my next PR if you want.